userguide/dailyuse/anbox.rst

```diff
@@ -65,6 +65,12 @@ You can check that adb server is correctly running locally on your phone by open
     phablet@ubuntu-phablet:~$ adb devices  
     List of devices attached  
     emulator-5558	device  
+    
+If you get ``emulator-5558  offline`` :
```

Comment:
Let's rephrase this to make it more clear: If adb fails with the error message blah...